game_eggs/minecraft/java/ftb/egg-ftb-modpacksch-server.json

```diff
@@ -69,7 +72,8 @@
             "default_value": "",
             "user_viewable": true,
             "user_editable": true,
-            "rules": "nullable|string|max:20"
+            "rules": "nullable|integer|min:1",
```

Comment:
you can not have it nullable and have a min value. then set it to required and specify a max value